packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -372,7 +372,7 @@ export const CallComposite = (props: CallCompositeProps): JSX.Element => {
       <BaseProvider {...props}>
         <CallAdapterProvider adapter={adapter}>
           <MainScreen
-            callInvitationUrl={callInvitationUrl}
+            callInvitationUrl={role ? undefined : callInvitationUrl}
```

Comment:
Is this needed? If Contoso wishes to provide a copyInviteLink should we not permit them to do so?

I definitely think we should have the below change where you only show the add people button iff there are actual menu items attached to it!

Reply:
(our samples and selfhost shouldn't pass it in though because it won't function correctly)

Reply:
Great point. We shouldn't hide a copyInviteLink if Contoso has provided one. Moved this logic to the Calling sample.